.gitmodules

```diff
@@ -1,6 +1,7 @@
 [submodule "src/llvm-project"]
 	path = src/llvm-project
-	url = https://github.com/llvm/llvm-project
+	url = https://github.com/dotnet/llvm-project
+	branch = dotnet/llvm-project
```

Comment:
do we need this? it looks like we get llvm from some transport package?

what about the git/config from savanna? and the wasi-libc?

Reply:
There are five pieces to the WASI SDK:

1. Build Clang from the llvm submodule
2. Build wasi-libc from the wasi-libc submodule using Clang from (1)
3. Build compiler-rt from the llvm submodule using Clang from (1)
4. Build libcxx from the the llvm submodule using Clang from (1)
5. Copy files around, including files from the config submodule.

The llvm transport package replaces step 1, but does not replace 3 and 4.

Reply:
We can't do 3 and 4 in `llvm-project` because we need the libc from `2` as a prerequisite, so our only real choice is to do it here - and we don't want to end up in an odd situation where we ship mismatched Clang binaries (from Dotnet LLVM 14) & headers (from upstream LLVM 15.0.7). Hence redirecting the submodule.